services/jitpack/jitpack-version.service.js

```diff
@@ -11,17 +11,16 @@ export default class JitPackVersion extends BaseJsonService {
   static category = 'version'
 
   static route = {
-    base: 'jitpack/v',
-    pattern: ':vcs(github|bitbucket|gitlab|gitee)/:user/:repo',
+    base: 'jitpack/version',
+    pattern: ':groupId/:artifactId',
```

Comment:
Lets leave a comment linking to https://github.com/badges/shields/issues/8312 to explain why we're using `/version` rather than `/v` here.